src/components/GlobalPreferences/Notifications/NotificationsLogin.js

```diff
@@ -0,0 +1,97 @@
+import React, { useState } from 'react'
+import styled from 'styled-components'
+import PropTypes from 'prop-types'
+import { Box, Button, GU, TextInput, Info, IconMail } from '@aragon/ui'
+import { login } from './notification-service-api'
+import { getEthNetworkType } from '../../../local-settings'
+import notificationSvg from './notifications.svg'
```

Comment:
@owisixseven We should also optimize this image; it's 2.3M.

Reply:
Added to to do list above so it doesn't get lost.